### DISCUSSION TITLE
Proposal: Introduce `learn.oqtane.org` for Documentation and Learning Resources

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.docs#101] [Ideas]
### Proposal: Introduce `learn.oqtane.org` for Documentation and Learning Resources  

Microsoft recently transitioned from `docs.microsoft.com` to `learn.microsoft.com`, signaling a shift toward a more comprehensive learning experience. Inspired by this, I propose introducing `learn.oqtane.org` as the new entry point for Oqtane's documentation and learning resources.  

By adopting "Learn" as the entry point, we can modernize our platform while retaining the existing top-level structure: "Guides," "User Manual," "Develop," and "API Documentation." This approach ensures minimal disruption while signaling a commitment to growth and inclusivity.  

#### Why `learn.oqtane.org`?  
- **Modernization:**  
   - Aligns with industry trends and positions Oqtane's resources as part of a modern learning ecosystem.  
- **Inclusivity:**  
   - The term "Learn" is approachable for users of all levels, creating a welcoming environment for beginners and advanced users alike.  
- **Flexibility for Growth:**  
   - Establishing `learn.oqtane.org` allows for potential future expansions, such as interactive tutorials or pathways for specific use cases.  

#### Proposal Highlights  
1. **Introduce `learn.oqtane.org`:**  
   Create a new subdomain (`learn.oqtane.org`) as the primary entry point to our documentation and educational resources.  

2. **Retain Current Navigation Structure:**  
   - Keep "Guides," "User Manual," "Develop," and "API Documentation" as the top-level navigation items. This minimizes disruption for users familiar with the current layout.  

3. **Explore Terminology Refinements:**  
   - While the current structure works well, we can consider future updates, such as enhancing "Guides" to include more practical "Tutorials" or reorganizing content for clarity.  

#### Proposed Steps  
1. **Set Up `learn.oqtane.org`:**  
   - Redirect `docs.oqtane.org` to `learn.oqtane.org` with appropriate DNS and server configurations.  
   - Ensure all existing `docs` URLs redirect seamlessly to the new subdomain.  

2. **Update Branding:**  
   - Update references to `docs.oqtane.org` in repositories, the site, and documentation headers to reflect the new subdomain.  
   - Add a transition banner:  
     _"Oqtane’s documentation has moved to `learn.oqtane.org`! Please update your bookmarks."_  

3. **Announce the Change:**  
   - Communicate the update through GitHub, social media, and newsletters to inform contributors and users.  

4. **Gather Feedback:**  
   - Monitor user feedback on the new URL and explore further refinements based on community input.  

#### Community Input  
I’d like to gather feedback from maintainers, contributors, and users:  
- Do you support the creation of `learn.oqtane.org` as the primary entry point for our documentation?  
- Are there additional considerations we should address during this transition?  

Looking forward to your thoughts and suggestions!

**Comment by iJungleboy** [comment]
At the moment I'm very hesitant to do anything like this.

Our docs are a lot more like the docs of thousands of open-source projects, and very little like the MS docs which has a team of probably 100+ people working on them full time. 

So I would really not invest into anything like this - IMHO it would just be different, but not better; just swapping some advantages for others.

So something we can do any time in future if we ever feel like it.
